man/systemd.exec.xml

```diff
@@ -824,7 +825,8 @@
         as a non-privileged user but still want to give it some capabilities.
         Note that in this case option <constant>keep-caps</constant> is
         automatically added to <varname>SecureBits=</varname> to retain the
-        capabilities over the user change.</para></listitem>
+        capabilities over the user change. AmbientCapabilities does not affect
```

Comment:
please write this as `<varname>AmbientCapabilities=</varname>`. i.e. always suffix with `=` if this is a setting, and enclose in the right docbook tag.